src/Components/ServiceScene/LogListControls.tsx

```diff
@@ -71,12 +76,24 @@ export const LogListControls = ({
       {showLabels !== undefined && onToggleLabelsClick && (
         <IconButton
           name="key-skeleton-alt"
+          aria-pressed={showLabels}
           className={showLabels ? styles.controlButtonActive : styles.controlButton}
           onClick={() => onToggleLabelsClick(!showLabels)}
           tooltip={showLabels ? 'Hide Labels' : 'Show labels'}
           size="lg"
         />
       )}
+
+      {showHighlight !== undefined && onToggleHighlightClick && (
+        <IconButton
+          name="brackets-curly"
+          aria-pressed={showHighlight}
+          className={showHighlight ? styles.controlButtonActive : styles.controlButton}
+          onClick={() => onToggleHighlightClick(!showHighlight)}
+          tooltip={showHighlight ? 'Disable highlighting' : 'Enable highlighting'}
+          size="lg"
+        />
+      )}
```

Comment:
Seems like we're overloading the button icons here with double meanings. In core the brackets are to collapse json, and a tag icon is used for labels. 
![Screenshot 2025-07-07 at 4 12 39 PM](https://github.com/user-attachments/assets/3aac86d8-2734-4111-a9d2-a7dbf052d02a)